src/AnimatedCounter.tsx

```diff
@@ -120,12 +120,13 @@ const NumberColumn = memo(({
     >
       <motion.div
         animate={{ x: 0, y: position }}
+        transition={{ duration: 0.3 }} // Adjust animation duration as needed
```

Comment:
With this addition, we'd lose the "bounce" effect that currently exists ([see style here](https://github.com/TuckerMassad/react-animated-counter/blob/main/src/styles.css#L31)). I'd suggest we remove this part of the PR and we have an additional PR open that introduces an `animationDuration` prop over here https://github.com/TuckerMassad/react-animated-counter/pull/11

Reply:
Alrighty